src/utils/formatTickPrice.ts

```diff
@@ -17,5 +17,5 @@ export function formatTickPrice(
   if (!price && placeholder !== undefined) {
     return placeholder
   }
-  return formatPrice(price, 6)
+  return formatPrice(price, 5)
```

Comment:
any specific reason?

Reply:
for consistent with left price and right price
lowest feetier has tick spacing 1 -> that means price move 0.0001 each tick